README.md

```diff
@@ -1 +1,678 @@
-# fixture-generator
\ No newline at end of file
+# Fixture Generator
```

Comment:
Nit: Personally, I like to see [examples](https://github.com/pallets/flask?tab=readme-ov-file#a-simple-example) of how to [use](https://github.com/typestack/class-validator?tab=readme-ov-file#usage) the code up-front, rather than delve into the history of why it needs to exist.

It's certainly nice to know the theory, but I kinda just want to know how to use the thing.